CIP-_/README.md

```diff
@@ -0,0 +1,209 @@
+---
+CIP: "?"
+Title: Optimized Coin Selection Algorithms for Cardano
+Category: Wallets
+Authors:
+    - Nicholas Maselli <maselli.nicholas@gmail.com>
+Implementors: CardanoSharp 7, SaturnNFT, Levvy Finance
+Discussions:
+Status: Pending
+Created: 2024-14-03
+License: CC-BY-4.0
+---
+
+## Abstract
+
+Coin Selection algorithms have become a critical part of the Cardano ecosystem. These algorithms choose a set of input Utxos for a given set of new Outputs.
+
+This CIP aims to present optimized coin selection algorithms to enable Wallets and Dapps to interact with smart contracts and native assets such as tokens and NFTs in a more efficient way. The goal of these algorithms is to primarily reduce the size of transactions as well as reduce the mem and step required for smart contract executions.
+
+## Motivation: why is this CIP necessary?
+
+<b>Cardano needs to scale.</b> As the ecosystem grows and new users come in, blocks are regular at 100% load with 88KB per block allocation. Recently, events such as the Lil Sappys NFT mint have pushed the Cardano blockchain load to over 100% for several hours, preventing anyone from using the chain and providing an awful user experience to new users who complain on social media and give the ecosystem a bad reputation.
+
+<b>This can't continue if Cardano is to succeed.</b>
```

Comment:
```suggestion
As the Cardano ecosystem grows and new users come in, blocks are regular at 100% load with 88KB per block allocation. Recently, events such as the Lil Sappys NFT mint have pushed the Cardano blockchain load to over 100% for several hours, preventing anyone from using the chain.
```
In general we need to avoid subjective & persuasive language because people cannot generally agree upon these statements at the present time.... let alone years down the road when this document still still persist while these arguments likely won't.

The Motivation is to outline technical imperatives... it's a courtesy to your readers & potential implementors to simply point out the facts that indicate what needs to be changed & allow them to draw their own conclusions about the extent & consequences of the problem.